HACKING.md

```diff
@@ -218,6 +218,13 @@ uuidgen -r > secret
 oc create secret generic github-webhook-shared-secret --from-file=secret
 ```
 
+### [PROD] Create quay.io image push secret
+
+This secret is used to push the resulting OCI image to Quay.io
+
+1. Obtain the file `oscontainer-secret` from BitWarden.
+2. Run: `$ oc create secret generic oscontainer-secret --from-file=dockercfg=oscontainer-secret`.
```

Comment:
OK, *really* final bits so we don't break the pipeline as soon as this merges:
1. Did you add the secret to BitWarden? I don't see it.
2. Did you already create the OpenShift secret? I don't see it either.

Reply:
Secret was added to BitWarden and OpenShift (both old and new cluster in case we rollback the migration temporarily).